documentation/plain_rules.md

```diff
@@ -96,7 +96,7 @@ The field `text` of the rule must be a valid EPL statement and additionally must
 expectations of perseo and perseo-core.
 
 EPL is documented in [Esper site](http://www.espertech.com/esper/esper-documentation), in particular
-[version 7.1.0](http://esper.espertech.com/release-7.1.0/esper-reference/html/index.html).
+[version 8.7.0](http://esper.espertech.com/release-8.7.0/reference-esper/html/).
```

Comment:
8.4.0 is the current version used by perseo-core

Reply:
However CNR in Perseo-Core says otherwise... https://github.com/telefonicaid/perseo-core/blob/master/CHANGES_NEXT_RELEASE

pom.xml uses 8.4.0, so I understand that Perseo-Core CNR should be fixed (along with this PR, of course)

Reply:
This PR fixed in da6a94e

Reply:
PR fixing Perseo-Core CNR https://github.com/telefonicaid/perseo-core/pull/179